install_rshim.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#Need to run this script using sudo
+
+# Install prerequisites
+echo "Installing prerequisites..."
+sudo tdnf install -y libusb-devel pciutils-devel
```

Comment:
`sudo`  not needed here.

Reply:
True, I changed the script to run only what is really necessary under sudo.